packages/formik-material-ui-pickers/rollup.config.js

```diff
@@ -1,9 +1,26 @@
 import typescript from 'rollup-plugin-typescript2';
 import pkg from './package.json';
 
-const external = ['react', 'react-native', 'formik', '@material-ui/pickers'];
+const external = [
+  'react',
+  'react-native',
+  'formik',
+  '@mui/lab/DatePicker',
```

Comment:
Since all pickers are now part of `@mui/lab`, I'm thinking maybe all these should go in `formik-material-ui-lab` and the package `formik-material-ui-pickers` should be abandoned...